### PR TITLE
adds ManiacCore

### PR DESCRIPTION
**Pull request by XDjackieXD**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,6 +55,15 @@
             <p>Some crazy guys writing mods for Minecraft.</p>
         </div>
     </div>
+    <div class="row">
+        <div class="column">
+            <h5>Projects</h5>
```
Comment (Kilobyte22) [inline index.html:60]: Is it smart to call this projects? This could disadvantage female projects.

File: index.html [hunk 2]
```diff
@@ -55,6 +55,15 @@
...
+	    <ul>
+		    <li><a href="https://github.com/TheManiacs/ManiacCore">Maniac Core</a> boilerplate & helpers for minecraft modding</li>
```
Comment (Kilobyte22) [inline index.html]: Please don't use `&` here

File: index.html [hunk 3]
```diff
@@ -55,6 +55,15 @@
...
+	    </ul>
+        </div>
+    </div>
+</div>
```
Comment (Kilobyte22) [inline index.html]: Why are there 2 divs on same indention level

**Review by Kilobyte22 — CHANGES_REQUESTED** [review]
Please fix

File: index.html (hunk 1 above)
Comment (XDjackieXD) [reply index.html:60]: yes. it's smart

File: index.html (hunk 2 above)
Comment (XDjackieXD) [reply index.html]: fixed

File: index.html (hunk 3 above)
Comment (XDjackieXD) [reply index.html]: fixed

**Review by Kilobyte22 — APPROVED** [review]

**Pull request merged**